package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@chakra-ui/icons": "^2.1.1",
+    "@chakra-ui/react": "^2.8.2",
```

Comment:
Once you address other comments, you should be able to remove this and just rely on `streetscape`.